code/modules/events/scrubber_clog.dm

```diff
@@ -0,0 +1,144 @@
+/datum/round_event_control/scrubber_clog
+	name = "Minor Scrubber Clog"
+	typepath = /datum/round_event/scrubber_clog
+	weight = 25
+	max_occurrences = 3
+	earliest_start = 5 MINUTES
+
+/datum/round_event/scrubber_clog
+	announceWhen = 1
+	startWhen = 10
+	endWhen = 6000
```

Comment:
All our events are very old and don't use the seconds / minutes defines, you should convert these over. 
```suggestion
	endWhen = 10 MINUTES
```